cmd/eval-dev-quality/cmd/evaluate_test.go

```diff
@@ -51,10 +55,17 @@ func TestEvaluateExecute(t *testing.T) {
 		Arguments: []string{
 			"--language", "golang",
 			"--model", "symflower/symbolic-execution",
-			"--repository", "golang/plain",
+			"--repository", "plain",
```

Comment:
Repositories should be relative to the testdata directory... so this should be "golang/plain" not "plain"

Reply:
already on it